generator.js

```diff
@@ -136,8 +136,8 @@ module.exports = class WebpackGenerator extends Generator {
 			.then(answer => {
 				if (answer) {
 					favPath = answer['favPath'];
-					this.options.env.configuration.dev.topScope.push('const FaviconsWebpackPlugin = require("favicons-webpack-plugin");');
-					this.dependencies.push("favicons-webpack-plugin");
+					this.options.env.configuration.dev.topScope.push("const WebappWebpackPlugin = require('webapp-webpack-plugin');");
```

Comment:
Can you please revert the `quotes` here ?

Reply:
Sure, In next commit